### ISSUE TITLE
It may not be working anymore

### ISSUE THREAD

**Issue by alanpt** [justinhunt/moodle-cloud9#1]
After running the Apache & PHP.. console says:

Starting Apache httpd, serving https://workspace-username.c9users.io
Started apache2

And then nothing happens, only the orange Welcome to the Cloud9 .... screen shows (even in the /moodle subdirectory)

**Comment by justinhunt** [comment]
I have used this very recently. So I think you can get it to work. The most common problem when cloning workspaces (not nec. directly from this script) is that the the URL of the Moodle site is not updated in Moodle config.php. Can you confirm that the Moodle site's config.php and the URL that your workspace is running on, match?

**Comment by alanpt** [comment]
Hi Justin,
The URLs match, This is what is found in the moodle config.php. The config.php in the root folder is the unedited file.

```
$CFG->dbtype    = 'mariadb';      // 'pgsql', 'mariadb', 'mysqli', 'mssql', 'sqlsrv' or 'oci'
$CFG->dblibrary = 'native';     // 'native' only at the moment
$CFG->dbhost    = 'localhost';  // eg 'localhost' or 'db.isp.com' or IP
$CFG->dbname    = 'c9';     // database name, eg moodle
$CFG->dbuser    = 'alanpt';   // your database username
$CFG->dbpass    = '';   // your database password

$CFG->wwwroot   = 'https://moodletesting-alanpt.c9users.io/moodle';
```

Your in NZ too,? I work at NMIT in Nelson.

**Comment by justinhunt** [comment]
OK It was a bit of a punt that the wwwroot was wrong. Since thats the step I always mess up when I clone a workspace. But since you posted the URL of your site I visited it. Then I clicked the "open this app" button, an orange screen opened up ... I waited a minute or so and boom up came the install Moodle page. Perhaps you were just being a hasty kiwi?
![2018-06-27_09-06-31](https://user-images.githubusercontent.com/1393905/41945832-1657c95c-79ea-11e8-913d-885428497ce0.png)
I am in Japan these days. But would love to be in Nelson. Love the Abel Tasman National Park
